Formula/neovim.rb

```diff
@@ -0,0 +1,90 @@
+class Neovim < Formula
+  desc "Ambitious Vim-fork focused on extensibility and agility"
+  homepage "https://neovim.io"
+
+  stable do
+    url "https://github.com/neovim/neovim/archive/v0.2.0.tar.gz"
+    sha256 "72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5"
+
+    resource "luarocks" do
+      url "https://github.com/luarocks/luarocks.git",
+        :revision => "5d8a16526573b36d5b22aa74866120c998466697"
```

Comment:
I just used the regular identation. Does this means that consecutive lines should use two "tab" stops ? The two tab stop spaces and the " align so both answers result in the same thing.